libs/service-logging/src/service_logging/middleware.clj

```diff
@@ -50,9 +50,10 @@
      (tc/with-logging-context {:response (cheshire/encode (clean-response (assoc response
                                                                            :uri uri
                                                                            :request-method request-method)))}
-                              (log/log 'AccessLogger level throwable (str method " " uri)))))
+                              (log/log 'AccessLogger level throwable (str method " " uri))
+                              (tc/clear-ext-svc-tag!))))
```

Comment:
I'm not quite sure why you did this the way you did -- a wrapping `with-logging-context` is already going to remove this at the end, basically no matter what (it'll either clear the context or replace it with whatever it had going in). The only time it'd be useful to explicitly clear this value is if you want a structure like

```clojure
(with-logging-context {...}
  (do some stuff)
  (set the external service tag somewhere, maybe somewhere nested)
  (do some more stuff)
  (clear tag)
  (do some more stuff explicitly without the external service tag context))
```

This seems questionable to me, and in any case could be accomplished with:

```clojure
(with-logging-context {...}
  (do some stuff)
  (with-logging-context {..., possibly empty}
    (set the external service tag somewhere, maybe somewhere nested)
    (do some more stuff))
  (do some more stuff explicitly without the external service tag context))
```